backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -62,8 +66,11 @@ public University updateUniversity(@PathVariable long id,
     //todo: Implement an endpoint to delete university (change active status)
 
     @GetMapping("/ambassadors")
-    public Iterable<Ambassador> listAmbassador() {
-        return ambassadorRepository.findAll();
+    public Iterable<Ambassador> listAmbassador(@RequestParam(defaultValue = "0",
+            required = false) int pageNumber,
+            @RequestParam(defaultValue = "4", required = false) int offset) {
```

Comment:
Don't recommend using a default value as users can get mislead into thinking there is only 4 records in the database

Reply:
@jayasanka-sack told it's better to use a default value

Reply:
As we don't have API documentation to let users know, my suggestion is to make this param required and add a min, max range to validate. You can do the same to the `pageNumber`. 
WDYT @jayasanka-sack

Reply:
Yeah I agree with your point. Let's make them required with a max-min validation.

Reply:
Ok I'm working on it.